aliases/available/general.aliases.bash

```diff
@@ -18,6 +18,11 @@ alias l1='ls -1'
 
 alias _="sudo"
 
+# Shortcuts to edit startup files
+alias vigen="vim ~/.bash_it/aliases/available/general.aliases.bash"
+alias vbrc="vim ~/.bashrc"
```

Comment:
I like this one! But similar to how the `reload` alias is defined, it might make sense to define one alias for these two commands (editing `~/.bashrc` and `~/.bash_profile`), depending on the OS. See https://github.com/Bash-it/bash-it/blob/master/bash_it.sh#L5 for an example.

That is unless you always want to provide the option of editing both files, regardless of the OS. In this case, it makes sense to add the two aliases.